specification/2.0/docinfo.html

```diff
@@ -0,0 +1,129 @@
+<style>
+/*
+ * Additional style rules for the glTF 2.0 Specification.
+ * Copyright (c) 2021 Khronos Group.
```

Comment:
Should be

> Copyright 2021 The Khronos Group Inc.

"(c)" is meaningless and the actual copyright symbol is redundant with "Copyright". Using full org name.

Reply:
Fixed in both docinfo.html and Specification.adoc.